src/app/Instance/CreateInstance/CreateInstance.test.tsx

```diff
@@ -115,13 +133,16 @@ describe("CreateInstance component", () => {
     await waitFor(() => comp.getByText("Webhook"));
     fireEvent.click(comp.getByText("Webhook"));
 
+    await act(async () => {
+      /* let component do its updates */
+    });
```

Comment:
This is waiting for all side effects to run. `fireEvent` should already be wrapped internally by an `act`, but for some reason some state updates are not being detected.

Reply:
isn't it an indicator of some 'bug'?

In the documentation I do not see explanation, when to use `act` with `empty` callback.

what component is `act` rendering in our test?

Why we do nto have sth like:
```js
act(() => {
    render(<ComplexSlowComponent name="RHOSE COMPONENT" />, container);
  });

Reply:
We don't need to wrap renders because it's already done by react testing library, [as stated in the docs](https://testing-library.com/docs/preact-testing-library/api/#act).

I've replaced anyway the obscure empty `act` with a `waitFor` that looks for the EH form field rendered after the schema loading/parsing. Same result but more clear.